checks/link-config.json

```diff
@@ -7,6 +7,14 @@
         "replacement": "{{BASEURL}}/"
       }
     ],
+    "httpHeaders": [
```

Comment:
As suggested on this [thread](https://github.com/tcort/markdown-link-check/issues/201)